src/App/frontend/src/layout/PaymentDetails/PaymentDetailsTable.module.css

```
@@ -6,6 +6,10 @@
   text-align: left;
 }
 
+.tableRow td {
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)